### DISCUSSION TITLE
Psycopg documentation is not clear about SQLA 2.0

### DISCUSSION THREAD

**Discussion by maybephilipp** [sqlalchemy/sqlalchemy#9858] [Usage Questions] [answered]
Documentation for psycopg (aka psycopg 3) here: https://docs.sqlalchemy.org/en/20/dialects/postgresql.html#module-sqlalchemy.dialects.postgresql.psycopg

It says:

> The psycopg dialect has the same API features as that of psycopg2, with the exception of the “fast executemany” helpers. The “fast executemany” helpers are expected to be generalized and ported to psycopg before the final release of SQLAlchemy 2.0, however.

SQLA 2.0 is released now, what is the current state of "fast executemany" support, who knows?

Is there any more cons in using version 3 of psyco?

Thanks:)

**Comment by CaselIT** [comment, marked as answer]
Hi,

That paragraph is outdated and should probably be removed.
I don't think there are any known limitations of psycopg3 at the moment.

**Reply by CaselIT** [reply]
I've removed it in a349447b7db45a3d5f08bc79eca225cec42b75d1
